lib/api/CAnomalyJob.cc

```diff
@@ -349,9 +349,7 @@ bool CAnomalyJob::handleControlMessage(const std::string& controlMessage) {
         this->doForecast(controlMessage);
         break;
     case 'w':
-        if (m_PersistenceManager != nullptr && this->isPersistenceNeeded("state")) {
-            m_PersistenceManager->startPersist(core::CTimeUtils::now());
-        }
+        this->doPersist(controlMessage.substr(1));
```

Comment:
I would rename `doPersist` to `processPersistControlMessage`.  We already have many similarly named methods related to persistence, so it would be good if the names of new ones make totally clear what they're for.